docs/components/databases.rst

```diff
@@ -102,10 +107,71 @@ How to use the :code:`mysql` or :code:`mariadb` format?
                 PYFUNCEBLE_DB_PORT=3306
                 PYFUNCEBLE_DB_USERNAME=pyfunceble
 
-4. Switch the :code:`db_type` index of your configuration file to :code:`mysql`
-   or :code:`mariadb`.
+4. Switch the :code:`db_type` index of your configuration file to
+   :code:`mariadb` or :code:`mysql`.
 5. Play with PyFunceble!
 
 .. note::
-    If the environment variables are not found, you will be asked to prompt the
-    information.
+    If the environment variables are not found, you will be asked to
+    prompt the information.
+
+SQL Layout:
```

Comment:
As of a possible merge of https://github.com/funilrys/PyFunceble/tree/issue-118 in the coming days, All INT (except for `http_status_code`) will be converted to BIGINT.

Reply:
I'll take note of that.

What about my question marks in the `SQL Layout:` L118-L177 ?

Another thing I just noticed, it the lines L73-L108 even relevant anymore after changing to sqlachemy?

Reply:
Let's get this one merged 

Then we polish it on the dev branch